mitodl/odl-video-service
# Enable anonymous access to collection pages

#### What are the relevant tickets?
Fix #706 

#### What's this PR do?
Allow anonymous user to view a collection page.

#### How should this be manually tested?
Create a collection with public and private videos. Try to view the collection detail page from an incognito window. You should be able to see the public videos.
